openkfw/TruBudget
# 1864 custom project images

### Checklist




### Description

Possibility too upload custom image

### How to test

1. login as e.g. mstein
2. Try to create new project
3. Upload custom image
4. Verify that project was created and image is stored and displayed




Closes #1864
